city_scrapers/spiders/cook_justice_advisory.py

```diff
@@ -42,7 +44,7 @@ def parse(self, response):
 
     def _parse_event(self, response):
         """Parse the event page."""
-        agenda_map = self._get_agenda()
+        agenda_map = self.agenda_map
```

Comment:
Is the `agenda_map` variable necessary here when we can call `self.agenda_map` directly?